examples/data_labeler/data_labeler_predict_train.ipynb

```diff
@@ -0,0 +1,370 @@
+    "        \"X-Origin: Smith-S\\n\" + \\\n",
+    "        \"X-FileName: SSMITH (Non-Privileged).pst\\n\\n\" + \\\n",
+    "        \"All I ever saw was the e-mail from the office.\\n\\n\" + \\\n",
+    "        \"Mary\\n\\n\" + \\\n",
+    "        \"-----Original Message-----\\n\" + \\\n",
+    "        \"From:   Smith, John  \\n\" + \\\n",
+    "        \"Sent:   Friday, August 10, 2005 13:07 PM\\n\" + \\\n",
+    "        \"To:     Smith, Mary W.\\n\" + \\\n",
+    "        \"Subject:        ABC\\n\\n\" + \\\n",
+    "        \"Have you heard any more regarding the ABC sale? I guess that means that \" + \\\n",
+    "        \"it's no big deal here, but you think they would have send something.\\n\\n\\n\" + \\\n",
+    "        \"John Smith\\n\" + \\\n",
+    "        \"123-456-7890\\n\"\n",
+    "\n",
+    "# convert string data to list to feed into data labeler\n",
+    "data = [data]\n",
+    "print(data[0])"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "geological-clark",
+   "metadata": {},
+   "source": [
+    "By default, data profiler predicts the results at the character level for unstructured text."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "theoretical-graph",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_labeler = dp.DataLabeler(labeler_type='unstructured')\n",
+    "\n",
+    "# make predictions and get labels per character\n",
+    "predictions = data_labeler.predict(data)\n",
+    "\n",
+    "# display results\n",
+    "print(predictions['pred'])"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "expensive-mechanics",
+   "metadata": {},
+   "source": [
+    "In addition to the character-level result, data profiler provides the results at the word level following the standard NER (Named Entity Recognition), e.g., utilized by spaCy. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "decreased-directive",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# convert prediction to word format and ner format\n",
+    "# Set the output to the NER format (start position, end position, label)\n",
+    "data_labeler.set_params(\n",
+    "    { 'postprocessor': { 'output_format':'ner', 'use_word_level_argmax':True } } \n",
+    ")\n",
+    "\n",
+    "# make predictions and get labels per character\n",
+    "predictions = data_labeler.predict(data)\n",
+    "\n",
+    "# display results\n",
+    "print('\\n')\n",
+    "print('=======================Prediction======================\\n')\n",
+    "for pred in predictions['pred'][0]:\n",
+    "    print('{}: {}'.format(data[0][pred[0]: pred[1]], pred[2]))\n",
+    "    print('--------------------------------------------------------')"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "narrow-census",
+   "metadata": {},
+   "source": [
+    "Data profiler identifies sensitive information such as datetime, email address, person names, and phone number. "
```

Comment:
I think you should have some kind of conclusion here. Like say all the capabilities we just learned or where you can go from here, etc.